Aquality.Selenium.Core/src/Aquality.Selenium.Core/Applications/Startup.cs

```diff
@@ -41,6 +41,7 @@ public void ConfigureServices(IServiceCollection services, Func<IServiceProvider
 
             services.AddTransient<IElementFinder, ElementFinder>();
             services.AddTransient<IElementFactory, ElementFactory>();
+            return services;
         }
 
         /// <summary>
```

Comment:
should the next method also be virtual?

Reply:
I think no, because this method gets default settings file. If we want to override it, we can override parameter 'ISettingsFile settings = null'.